### PR TITLE
Fix localizations

### PR DESCRIPTION
**Pull request by jasonLaster**
### Summary of Changes

Fixes issues found in #4932

### CODE REVIEW

File: assets/panel/debugger.properties [hunk 1]
```diff
@@ -707,7 +707,7 @@ functionSearchSeparatorLabel=←
 # Do not localize "CmdOrCtrl+;", or change the format of the string. These are
 # key identifiers, not messages displayed to the user.
 gotoLineModal.placeholder=Go to line…
-gotoLineModal.key=CmdOrCtrl+;
+gotoLineModal.key2=CmdOrCtrl+;
```
Comment (flodolo) [inline assets/panel/debugger.properties]: Can we split the part of comment about the shortcut into a separate comment? Right now it's on the placeholder string's comment.

**Comment by jasonLaster** [comment]
fixed. @flodolo  what do you think?

**Comment by juliandescottes** [comment]
I don't think the comment from @flodolo was actually addressed here? The idea was to move the comment right above `gotoLineModal.key2=CmdOrCtrl+;` if I read that correctly.

Also this is not included in release 15. @jasonLaster we should fix this asap, can you cherry pick this one in the release?

**Comment by flodolo** [comment]
> Also this is not included in release 15. @jasonLaster we should fix this asap, can you cherry pick this one in the release?

I'm confused, I'm sure I've seen a version of this with the comment split (there's about an hour between my comment and the second one).

**Comment by flodolo** [comment]
> I'm confused, I'm sure I've seen a version of this with the comment split (there's about an hour between my comment and the second one).

OK, I'm not crazy. Here was [the commit](https://github.com/devtools-html/debugger.html/commit/f9209bc) I've approved, resurrected from my emails. No clue what happened in-between.

**Comment by juliandescottes** [comment]
I would guess that a `-force` push erased the commit you reviewed and it was the one that got merged into master?

**Comment by flodolo** [comment]
It's still listed above the merge. 

screenshot-2018-2-9 fix localizations by jasonlaster pull request 5360 devtools-html debugger html

As far as I've seen, a force push would hide it from the PR, while the commit itself would be still reachable (that's why I went straight to my email).

**Comment by jasonLaster** [comment]
got it. I'll push a fix.

re: release 15, this commit landed after that was cut. I'll get 16 out soon

**Pull request merged**